modules/con-sdpolicy-account-management.adoc

```diff
@@ -6,48 +6,37 @@
 = Account management
 
 == Billing
-Each {product-title} cluster requires a minimum annual base cluster purchase and there are two billing options available for each cluster, Standard and Customer Cloud Subscription (CCS).
+{product-title} is billed through Amazon Web Services (AWS) based on usage of AWS components used by the service, such as load balancers, storage, EC2 instances, and so on.
```

Comment:
> ! and so on n
Use only when you list a clear sequence of elements, such as “1, 2, 3, and so on” or “Monday, Tuesday, Wednesday, and so on.” When you list items that do not form a clear sequence, use appropriate descriptive wording, such as “device drivers, firmware, and other code.”

Reply:
Fixed in the new PR.